lib/puppet-lint/plugins/legacy_facts/legacy_facts.rb

```diff
@@ -0,0 +1,189 @@
+# Public: A puppet-lint custom check to detect legacy facts.
+#
+# This check will optionally convert from legacy facts like $::operatingsystem
+# or legacy hashed facts like $facts['operatingsystem'] to the
+# new structured facts like $facts['os']['name'].
+#
+# This plugin was adoped in to puppet-lint from https://github.com/mmckinst/puppet-lint-legacy_facts-check
```

Comment:
```suggestion
# This plugin was adopted in to puppet-lint from https://github.com/mmckinst/puppet-lint-legacy_facts-check
```

Reply:
What's funny is that I noticed this when I copied the line from the other check.. with full intention of fixing both errors... then forgot and committed it anyway. 🤷

I'll get that sorted now.